pkg/volume/empty_dir/empty_dir.go

```diff
@@ -254,8 +255,36 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 		return nil
 	}
 
-	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v", ed.pod.UID, ed.volName)
-	return ed.mounter.Mount("tmpfs", dir, "tmpfs", nil /* options */)
+	memoryLimits, err := ed.getMemoryLimits()
+	if err != nil {
+		return err
+	}
+	var options []string
+	if memoryLimits > 0 {
+		options = []string{fmt.Sprintf("size=%d", memoryLimits)}
```

Comment:
what does linux default size to, if not specified?  Maybe comment all this to that effect